src/libraries/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -316,6 +316,7 @@ public static string[] GetNames(Type enumType)
             // Get all of the names
             GetEnumInfo(enumType).Names;
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This is fairly small method. Does it really need `AggressiveInlining`?

Reply:
Actually haven't checked, just put it here to ensure it's inlined for now.